styles/styles.css

```diff
@@ -380,3 +382,104 @@ main .section.light,
 main .section.highlight {
   background-color: var(--color-neutral-500);
 }
+
+/* Base Design System Tokens */
+.elsie-base-design.elsie-base-design {
+  --background-none: none 0 0% no-repeat padding-box;
+}
+
+/* de-normalizing */
+.elsie-base-design button[class^="elsie-incrementer__"],
+.elsie-base-design input[class^="elsie-incrementer__"] {
+  margin: unset;
+  padding: unset;
+}
+
+.elsie-base-design h1, .elsie-base-design h2, .elsie-base-design h3,
+.elsie-base-design h4, .elsie-base-design h5, .elsie-base-design h6 {
+  margin: 0;
+  padding: 0;
+}
+
+.elsie-base-design hr {
+  border-color: var(--color-neutral-400);
+  margin: 16px 0;
+}
+
+.elsie-base-design ul {
+  margin-top: 0;
+}
+
+.elsie-base-design button {
+  border-radius: 4px;
+  color: var(--color-neutral-50);
+  font-size: 12px;
+  line-height: 1.25;
+  margin: 0;
+  padding: var(--spacing-small) var(--spacing-medium);
+  text-transform: none;
+}
+
+.elsie-base-design .elsie-button--tertiary {
+  color: var(--color-brand-500);
+}
+
+.elsie-base-design h2 {
+  font: var(--type-body-1-default-font);
+  font-weight: 600;
+  letter-spacing: .2em;
+  line-height: 1.5;
+}
```

Comment:
I'm not sure about the styles included in this file, but some of them are overriding the styling from our components & some others are not using SDK tokens

Reply:
I think those were added as part of the PDP drop-in. @alandana are those required or could we fallback to out of the box styling of the PDP drop-in using tokens only?

Reply:
I added those when we were working on the Asprey implementation and there was a clash between the EDS styling and the components' styling. I believe this can be removed now.

Reply:
I could not remove all of the styles, but I scoped the ones that are required to the PDP. So they should no longer affect the checkout. See https://github.com/hlxsites/aem-boilerplate-commerce/pull/3/commits/af632c0f9f1c5490f1cb4a65dc371f4c7e4e39d4.

Reply:
I've spotted 2 of the main CSS styles that affect the checkout directly: 
- `main input` (line 310) overrides margin and padding (not using tokens)
- The normalizing styles for headlines (line 398)

Let me know if we can remove them safely without breaking PDP